politician_analysis/src/scraper.py

```diff
@@ -0,0 +1,126 @@
+import json
+import time
+import requests
+
+
+YOUTUBE_URL = 'https://www.youtube.com/watch?v={youtube_id}'
+USER_AGENT = 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.130 Safari/537.36'
+SORT_BY_POPULAR = 0
+SORT_BY_RECENT = 1
+
+def find_value(html, key, num_chars=2, separator='"'):
```

Comment:
I look at the following example of calling this function, and most of them do not use the default value of the parameter. I suggest that we can delete the default value.

Reply:
got it